tests/models/whisper/test_modeling_whisper.py

```diff
@@ -1912,6 +1912,177 @@ def test_tiny_timestamp_generation(self):
         transcript = processor.batch_decode(generated_ids, skip_special_tokens=True, output_offsets=True)
         self.assertEqual(transcript, EXPECTED_TRANSCRIPT)
 
+    @slow
+    def test_large_timestamp_generation(self):
+        set_seed(0)
+        processor = WhisperProcessor.from_pretrained("openai/whisper-large-v3")
+        model = WhisperForConditionalGeneration.from_pretrained("openai/whisper-large-v3")
+        model.to(torch_device)
+
+        input_speech = np.concatenate(self._load_datasamples(4))
+        input_features = processor(
+            input_speech, return_tensors="pt", sampling_rate=16_000, return_token_timestamps=True
+        ).input_features
+        input_features = input_features.to(torch_device)
+
+        generated_ids = model.generate(input_features, max_length=448, return_timestamps=True).to("cpu")
+
+        EXPECTED_OUTPUT = torch.tensor(
```

Comment:
Can you put the expected outputs on one line, and add ` # fmt: skip ` at the end?
```
EXPECTED_OUTPUT = torch.tensor([ 50258, ..., 50257]) # fmt: skip
```

E.g. as done here:
https://github.com/huggingface/transformers/blob/f4014e75db0190792b3feeccfc5dc5b5f9f0ce7b/tests/models/whisper/test_modeling_whisper.py#L1903

This will stop the linter from formatting it over hundreds of lines, thus improving readability of the test